styles.css

```diff
@@ -9,6 +9,29 @@
   margin: 50px auto;
 }
 
-p {
+
+p, ol {
+  font-family: fantasy;
+  font-size: large;
+  word-spacing: 5px;
+  line-height: 2em;
+}
+
+hr{
```

Comment:
the color is a dark blue, i am not sure if this is the right color.  suggestions??